src/server/services/config.service.ts

```diff
@@ -0,0 +1,59 @@
+import { MongooseModuleOptions } from '@nestjs/mongoose';
+
+/**
+ * Parses process.env to create a clean configuration interface
+ */
+
+class ConfigService {
```

Comment:
You've probably already seen this - but what do you think of using [config validation](https://docs.nestjs.com/techniques/configuration#validation)?

Reply:
Per our offline discussion, we'll consider this as a future enhancement but not as part of the pr.